src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,77 @@
+.nav {
+  visibility: visible;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    height: 60px;
+
+    list-style: none;
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    padding: 23px 0;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #060b35;
+    transition: $transition;
+
+    &::after {
+      content: "";
+
+      height: 4px;
+      position: absolute;
+      bottom: 0;
+      right: 0;
+      left: 0;
+
+      visibility: hidden;
+      background-color: $primary-color;
+      border-radius: 8px;
+    }
+
+    &:hover,
+    &:active {
+      color: $primary-color;
+    }
+
+    &:active::after {
+      visibility: visible;
+    }
+
+  }
+}
+
+/* The is-active class link */
+
+.is-active {
+  color: $primary-color;
+
+  &::after {
+    content: "";
+
+    height: 4px;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    visibility: visible;
```

Comment:
You can achieve this without using visibility

Reply:
Fixed